Makefile

```diff
@@ -241,7 +241,6 @@ dependencies: \
 
 ## Run cmd/addon-operator-manager against $KUBECONFIG.
 run-addon-operator-manager:
-.PHONY: run-addon-operator-manager
```

Comment:
Shouldn't that stay?

Reply:
lol YES!!

Reply:
Actually NO!

with `.PHONY` in place:
```
$ make run-addon-operator-manager
make: Nothing to be done for 'run-addon-operator-manager'.
```

but i should split this fix out of this PR to keep the fix separate from feature-work

Reply:
keeping it now since you already reviewed but I promise to spilt my PRs better in the future (also avoiding the thing that i did here: mixing test-code cleanup into a feature PR)